central/main.go

```diff
@@ -837,9 +838,12 @@ func customRoutes() (customRoutes []routes.CustomRoute) {
 		routes.CustomRoute{
 			Route: scannerDefinitionsRoute,
 			Authorizer: perrpc.FromMap(map[authz.Authorizer][]string{
-				or.SensorOr(
-					or.ScannerOr(
-						user.With(permissions.View(resources.Administration)))): {
+				or.Or(
+					idcheck.SensorsOnly(),
+					idcheck.ScannerOnly(),
+					idcheck.ScannerV4IndexerOnly(),
```

Comment:
Should we add matcher also? Since matcher will reach out to api for CVSS data

Reply:
good point